characters/charactersRouter.js

```diff
@@ -11,11 +11,13 @@ router
     .route('/')
     .get((req, res) => {
         if (req.query) {
-            Character.find({ $and: [{ height: { $gte: 100 } }, { gender: "female" }] })
+
+            Character.find({ gender: 'female', height: { $exists: true }, $where: `this.height >= 100` })
+                // { $and: [{ height: { $gte: 100 } }, { gender: "female" }] }
```

Comment:
Try to change the height in the schema from String to Number, and you should be good. ```{ $and: [{ height: { $gte: 100 } }, { gender: "female" }] }```